src/patterns/data/matches_annotations/location_top.tsv

```diff
@@ -1,73 +1 @@
 defined_class	defined_class_label	location	location_label	location_match_property

```

Comment:
@nicolevasilevsky Why do you think location_top matches have failed? Did the pattern change recently?

Reply:
hmmm, I don't know. I don't think it has changed, and spot checking a few terms, it looks like they conform to the pattern.